src/Compatibility/Core/src/iOS/Extensions/CompatibilityColorExtensions.cs

```diff
@@ -20,17 +20,6 @@ public static UIColor FromPatternImageFromBundle(string bgImage)
 
 			return UIColor.FromPatternImage(image);
 		}
-
-		public static UIColor ToUIColor(this Color color)
```

Comment:
should we obsolete these instead of delete?

Reply:
Not sure. I figured since we have a lot of breaking changes anyway let's just make this one more...?

Reply:
i m with @jfversluis  on this one

Reply:
I think our API surface hasn't broke too much though. We could probably mark these obsolete and guide users to `ToPlatform`? Just have the implementation on these call `ToPlatform` as well

Reply:
Well, even the name of this class has changed: https://github.com/dotnet/maui/pull/5854/files#diff-299e38c15e99c193ea7431ba10b5c87d2d234842badeb41c0215afa0cba437f8R12 😬 

Happy to make them obsolete first though. However, changing the implementation to `ToPlatform` will require to add a reference to Maui.Core to Compatibility, not sure if that's something we want?

Reply:
Did it for you Shane! Looks like my earlier attempt was just giving me false errors. Should be all good now!